src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -69,6 +69,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	private _content: string | string[];
 	private _lastTrustedMode: boolean;
 	private isEditMode: boolean;
+	private doShowPreview: boolean;
```

Comment:
Typical naming conventions suggest that having an action like `do` in a name means that it actually does something - so this would be more appropriate on a function that actually has direct side effects. 

I'd just name this `showPreview` (and then also update it to a property getter/setter as I mention below)

Reply:
Renamed to showPreview and added get/set. Need help wiring these up correctly.